charts/jaeger/templates/es-index-cleaner-cronjob.yaml

```diff
@@ -54,7 +54,6 @@ spec:
               - {{ .Values.esIndexCleaner.numberOfDays | quote }}
               - {{ include "elasticsearch.client.url" . }}
               {{ include "extra.cmdArgs" ( dict "cmdlineParams" .Values.esIndexCleaner.cmdlineParams ) | nindent 14  }}
-              {{- include "storage.cmdArgs" . | nindent 14 }}
```

Comment:
What if someone is already using `storage.cmdArgs`? This would introduce a breaking change for them, wouldn't it? I guess this is the reason it was left like this.

Reply:
This line was added last month to fix the issue #329.
But it shouldn't be there at all. Adding esIndexCleaner.cmdlineParams should have been the fix.

as you can see in the logs of #424 the storage args are different from the index cleaner args, which causes the index cleaner to stop working at all